.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["3.1", "3.0", "2.7", "2.6"]
+        ruby-version: ["3.1", "3.0"]
```

Comment:
giving it some more thought now...
this gem is not only ours, we've actually published it to rubygems so it fells wrong to simply break its compatibility with previous Ruby version for no reason

Reply:
hey @ajGingrich , have you ever re-generated the Buildium SDK? asking to know if you have the operan-api-generator set up in your machine

Reply:
I'm afraid we are causing too many different and big changes when only wanted to re-generate the SDK but got blocked by the openapi-generator CLI failing on both Leo's and my machines :/

Reply:
I've never generated any of the integrations SDKs 😢

Reply:
I think I have done that before [here](https://github.com/LeadSimple/buildium/pull/6), that was in my first few months here, I can't completely remember how I did it though but I didn't have this many changes. Would you want me to try something?

Reply:
Fixed guys.